exercises/accumulate/example.jl

```diff
@@ -0,0 +1,23 @@
+"""
+    accumulate(collection, operation)
+Takes in a collection and applies the operation on that collection then returns
+a new collection.
+"""
+function accumulate(collection, operation)
+
+    new_colletion = []
+
+    if operation == *
+        new_colletion = collection .* collection
+    elseif operation == +
+        new_colletion = collection .+ collection
+    elseif operation == -
+        new_colletion = collection .- collection
+    elseif operation == /
+        new_colletion = collection ./ collection
+    elseif operation == %
+        new_colletion = collection .% collection
+    end
```

Comment:
Untested, but I think this can be simplified to

```julia
new_collection = operation.(collection, collection)
```

Reply:
Yes, this is the better way indeed